lib/bloc/lnurl/lnurl_bloc.dart

```diff
@@ -46,18 +46,23 @@ class LNUrlBloc with AsyncActionsHandler {
         injector.nfc.receivedLnLinks(),
         injector.lightningLinks.linksNotifications,
         _lnurlInputController.stream,
-
-        // Process lightning-addresses in the clipboard only when the user manually 
-        // pastes them in and they show up in _lnurlInputController.stream.
-        injector.device.distinctClipboardStream
-            .where((l) => !isLightningAddress(l)),
+        injector.device.distinctClipboardStream.where((l) {
+          var result = true;
+          if (isLightningAddress(l)) {
+            result = false;
```

Comment:
This line is not needed.

Reply:
The redundancy is just to document the intention that we should not process email addresses in the clipboard. In case more logic was ever added to or changed in that block, the initial intention would be clear.

Reply:
Regarding documentation Isn't it better to add a comment instead? I find it confusing to see the variable assigned with a value that is overridden a line after.

Reply:
> Regarding documentation Isn't it better to add a comment instead?

Okay. Thanks. I'll do that.